Formula/creduce.rb

```diff
@@ -73,6 +72,14 @@ def install
       end
     end
 
+    # Work around build failure seen on Apple Clang 13.1.6 by using LLVM Clang
+    # Undefined symbols for architecture x86_64:
+    #   "std::__1::basic_stringbuf<char, std::__1::char_traits<char>, ...
+    if DevelopmentTools.clang_build_version == 1316
+      ENV["CC"] = Formula["llvm@9"].opt_bin/"clang"
+      ENV["CXX"] = Formula["llvm@9"].opt_bin/"clang++"
+    end
```

Comment:
Not sure what is best way to deal with undefined symbols. Looks like this didn't happen on previous Monterey's Apple Clang. Not sure if only impacts 13.1.6 or is going to be seen on later versions.

Problem with `fails_with :clang ...` is that `llvm@...` formulae are not considered valid compilers by brew and this will force an unnecessary `llvm` or `gcc` dependency.